universal-application-tool-0.0.1/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -57,10 +63,34 @@ public Result dashboard(Http.Request request) {
     if (trustedIntermediaryGroup.isEmpty()) {
       return notFound();
     }
+    ImmutableList<Account> managedAccounts =
+        trustedIntermediaryGroup.get().getManagedAccounts(search);
+    int endOfListIndex = page.get() * PAGE_SIZE;
+    boolean hasNextTab = true;
+    if (managedAccounts.size() <= page.get() * PAGE_SIZE) {
+      endOfListIndex = managedAccounts.size();
+      hasNextTab = false;
+    }
+    if (managedAccounts.size() <= (page.get() - 1) * PAGE_SIZE) {
```

Comment:
It'd probably be useful to calculate the number of pages instead of infer them here. Then in the UI we can either show **Page 2 of 14** or **Results 11 - 20 of 135**

Reply:
Done!
<img width="981" alt="Screen Shot 2021-05-25 at 11 25 27 AM" src="https://user-images.githubusercontent.com/605324/119549365-fbd26a80-bd4b-11eb-874f-409a016b4664.png">
<img width="972" alt="Screen Shot 2021-05-25 at 11 25 35 AM" src="https://user-images.githubusercontent.com/605324/119549373-ff65f180-bd4b-11eb-994b-72a4b2b2a481.png">